src/app/clusters/bindings/bindings.h

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app/clusters/bindings/BindingManager.h>
+#include <app/util/binding-table.h>
+
+/**
+ * @brief Saves binding in storage and notifies a new binding has been added
+ *
+ * @param entry binding to save
+ */
+void SaveBindingEntry(EmberBindingTableEntry & entry);
```

Comment:
This should really be `AddBindingEntry`, documented as adding this entry to the binding table and doing whatever persistence and notification work is involved in that.

Reply:
Added more specifc comments on what the function does.